Lib/defcon/objects/font.py

```diff
@@ -138,13 +138,13 @@ def __init__(self, path=None,
         self._dirty = False
 
         if path:
-            reader = UFOReader(self._path)
+            reader = UFOReader(self._path, validate=self.ufoLibReadValidate)
```

Comment:
I think it would be convenient if the Font constructor had a `validate` optional argument (that is stored as `self.ufoLibReadValidate`), and similarly for the `Font.save` method.

Reply:
alright